src/exberry_int/integration_exberry.py

```diff
@@ -186,75 +176,70 @@ async def handle_create_instrument_request(event):
                     logging.warning(f"Unknown response ¯\\_(ツ)_/¯ : {json_resp}")
 
 
-    @events.time.periodic_interval(env.interval)
-    async def process_inbound_messages():
-        if not inbound_queue.empty():
-            LOG.info(f"Will process {inbound_queue.qsize()} messages...")
-
+    @events.queue.message()
+    async def process_inbound_messages(msg):
         commands = []
```

Comment:
looks like after this change you are either returning a single command or nothing. Maybe it is better to not use a list and append a single element to it?